### DISCUSSION TITLE
怎么获取到 Modal 组件的 contentPadding 属性

### DISCUSSION THREAD

**Discussion by ekuku1984** [ant-design/ant-design#46528] [General]
怎么获取到 Modal 组件的 contentPadding  token 值。

使用
import { theme } = antd;
const { token } = theme.useToken();

token 里没有找到 contentPadding

**Comment by Wxh16144** [comment]
取不到，这一个内部的变量： https://github.com/ant-design/ant-design/blob/ce28f36adba03bcc77b8ad03f77576ad61afb4fd/components/modal/style/index.tsx#L379-L381

可以直接参考这个，自己写

**Reply by jiajiaxing1** [reply]
赞
